lib/src/sv_onvif.h

```diff
@@ -54,6 +54,12 @@ MediaSigningReturnCode
 onvif_media_signing_set_max_signing_frames(onvif_media_signing_t *self,
     unsigned max_signing_frames);
 
+MediaSigningReturnCode
+onvif_media_signing_set_use_certificate_sei(onvif_media_signing_t *self, bool enable);
+
+MediaSigningReturnCode
+onvif_media_signing_generate_certificate_sei(onvif_media_signing_t *self);
+
```

Comment:
Can we try to keep the definitions in sv_onvif.c in the same order as they are declared in sv_onvif.h